components/Box.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
I don't think we should create a "Box" component. If we end up using it, `Box` will just become overloaded to the point of it being equally useful as `div`.

If you want to use default values for margins, use the CSS variables found in: `components/style/Typography.js`